### PR TITLE
Ch930 sort by column

### PR DESCRIPTION
**Pull request by rpiaggio**

### CODE REVIEW

File: common/src/main/webapp/less/style.less [hunk 1]
```diff
@@ -34,6 +34,10 @@ body {
   width: 100vw;
 }
 
+a {
+  color: lightblue;
```
Comment (cquiroz) [inline common/src/main/webapp/less/style.less:38]: Do we want to change every link? At any rate this should be a `var`

**Review by cquiroz — APPROVED** [review]
Cool, do you have a screenshot

File: explore/src/main/scala/explore/targeteditor/TargetSummaryTable.scala [hunk 2]
```diff
@@ -125,20 +135,95 @@ object TargetSummaryTable {
...
               ).rawElement
           )
-          .setHeader("Observations")
-      ).toJSArray
+        ).toJSArray
+
+      // // All this is a hack while we don't have proper hooks.
+      val rawData = props.pointingsWithObs.targets.toList.toJSArray
+
+      // val reuseBy = (
```
[the remark this answers is not in the document]
Comment (rpiaggio) [reply explore/src/main/scala/explore/targeteditor/TargetSummaryTable.scala:143]: yup, again... let me remove that...

File: common/src/main/webapp/less/style.less (hunk 1 above)
Comment (rpiaggio) [reply common/src/main/webapp/less/style.less:38]: yes, agreed, I'm not familiar with how theming works and vars, let me check

**Comment by rpiaggio** [comment]
Screenshot, with column selection too:

image

**Comment by cquiroz** [comment]
Looks great

**Pull request merged**